src/Chapter3.hs

```diff
@@ -910,6 +992,20 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold Int
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append (Gold n) (Gold m) = Gold (n + m)
+
+instance Append [a] where
+  append :: [a] -> [a] -> [a]
+  append xs ys = xs ++ ys
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```

Reply:
I know that eta-reduce can be applied in this case as well, I learned a lot.